### DISCUSSION TITLE
Failed To Set Thread Priority

### DISCUSSION THREAD

**Discussion by punksinatra** [mbround18/valheim-docker#332] [Q&A] [answered]
My server is running fine but whenever I restart or start my server always get these lines on the log and was wondering what could be causing this or is this thread priority for

==> /home/steam/valheim/logs/valheim_server.log <==,
Warning: failed to set thread priority: set failed for 8: -1: setpriority() failed,
Warning: failed to set thread priority: set failed for priority 8,
Warning: support for elevated priorities is most likely unavailable, suppressing future warnings,

**Comment by mbround18** [comment, marked as answer]
Are you running the container privileged? Or using some sort of funky permissions? what you could try and do is remove the `/home/steam/valheim` folder and let Odin redownload the server files.

**Reply by punksinatra** [reply]
Yeah I am running it as a user not as root.. should it make any difference? Was wondering what thread priority means in this case

**Reply by mbround18** [reply]
I think it is related to your host OS and host system, what kind of system are you using for a host?

**Reply by punksinatra** [reply]
Debian 10 Buster

**Reply by mbround18** [reply]
Ill spin up a system and see if i get the same thing tomorrow <3

**Reply by punksinatra** [reply]
Thanks.. will wait for info from your end

**Reply by mbround18** [reply]
Hey @punksinatra apologies for the delay! Just got my dev machine fixed up and CI in the repo repaired. I'll deploy to a Debian instance tomorrow and give it a whirl :)

**Reply by punksinatra** [reply]
No worries mate. Will check in with you tomorrow. Thanks for the help appreciate it

**Reply by mbround18** [reply]
Alright i have an hourly instance of debian 10, installing docker and the container right now.

**Reply by mbround18** [reply]
@punksinatra I did some digging, i did not get the same error on my side
https://asciinema.org/a/V4RLpgOX16Kae15EbEMjt8EeS

With the research, it looks like its an issue between the processor, graphics card, & steamcmd combined. Its not a worry or an issue but more or an annoyance. What i can try doing is ensuring the docker image has an updated instance of the steam cmd docker image dependency but other than that it might just be always present. Has it caused any issues for you when your running your server?

**Reply by punksinatra** [reply]
Thanks @mbround18 for checking. I guess the hardware you are trying it on is a powerful one? It has not really caused anything but you are correct it's a bit annoying when I check logs. 

Thank you for checking :)

**Reply by mbround18** [reply]
Nah, I was using OVH sandbox for a cloud vps. I think it boils down to the environment its running in nd all :P
